shapecorners_config.ui

```diff
@@ -96,13 +103,20 @@
             </property>
            </widget>
           </item>
-          <item row="0" column="1">
-           <widget class="KColorButton" name="outlineColor">
-            <property name="alphaChannelEnabled">
-             <bool>true</bool>
+          <item row="1" column="0">
+           <widget class="QLabel" name="inactiveOutlineColorLabel">
+            <property name="text">
+             <string>Inactive Outline Color</string>
```

Comment:
Now that we are separating the outline colour for active and inactive windows, I suggest keeping both labels consistent and mentioning the term "Active" in the other label. Or whatever you feel is more understandable for someone who is seeing the dialog for the first time.

Reply:
I made both labels a bit longer, but they are consistent now. Thank you for noticing this!